assets/src/scripts/components/dailyValueHydrograph/selectors/scales.js

```diff
@@ -9,19 +9,22 @@ import {
 
 import {getLayout} from './layout';
 
+/*
+ * Returns a selector function which returns a d3 scale for either the brush,
+ * kind = 'BRUSH' or the main xscale for the dv graph.
+ */
 export const getXScale = memoize((kind) =>createSelector(
     getLayout(kind),
     getCurrentObservationsTimeSeriesTimeRange,
     state => state.observationsState.dvGraphBrushOffset,
     (layout, timeRange,dvGraphBrushOffset) => {
-        let xScale = scaleLinear();
-        if (timeRange) {
-            xScale
-                .range([0, layout.width - layout.margin.right])
-                .domain([timeRange.startTime, timeRange.endTime]);
-            if (dvGraphBrushOffset) {
-                xScale.domain([timeRange.startTime + dvGraphBrushOffset.start, timeRange.endTime - dvGraphBrushOffset.end]);
-            }
+        let xScale = scaleLinear()
+            .range([0, layout.width - layout.margin.right]);
+
+        if (kind !== 'BRUSH' && timeRange && dvGraphBrushOffset) {
```

Comment:
This is curious:
if (kind !== 'BRUSH'
shouldn't the kind == Brush here?

Reply:
The first if clause is for the Main XScale which should be modified with the brush offset when that  and timeRange are available. The next else clause says to use the time range if available to set the scale range. If we have a BRUSH scale it will fail the first if and take this one or the last (if the data has not yet been fetched).